### PR TITLE
Update print_to_gt.R to put RTF footnotes in the document footer by default 

### LINKED ISSUE #422: put RTF footnotes in the document footer by default
Add this to the existing series of `tab_options` within the `print_to_gt` function: `tab_options(page.footer.use_tbl_notes = TRUE)`

This will get picked up if gt exports the table to an RTF document, but ignored otherwise.

### PR DESCRIPTION
**Pull request by alanahjonas95**
resolves #422

### CODE REVIEW

**Comment by bzkrouse** [comment]
@alanahjonas95 Thanks!! Can you please check this does what we want for saving to RTF, and doesn't impact other outputs? You could use {tfrmtbuilder} to make this quicker!

**Comment by alanahjonas95** [comment]
@bzkrouse just made this quickly with tfrmtbuilder as suggested :)
looks like this with rtf now:
image
but all other methods appear with it just underneath eg:
this for png
image
and this for docx
image

**Comment by bzkrouse** [comment]
@alanahjonas95 Perfect thank you!! One more thing, can you please add to the NEWS file ?

**Comment by alanahjonas95** [comment]
> @alanahjonas95 Perfect thank you!! One more thing, can you please add to the NEWS file ? Then we should be good to go.

I've updated the read me but wasn't sure if i should mark as a bug fix or an improvement? and will the next release be 0.1.4?

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # tfrmt development version
 
+# tfrmt 0.1.4
```
Comment (bzkrouse) [inline NEWS.md:3]:
```suggestion 
```

Comment (bzkrouse) [reply NEWS.md:3]: we can just nest under "tfrmt development version" header for now :)

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+
+Bug fixes
```
Comment (bzkrouse) [inline NEWS.md:5]:
```suggestion
Improvements
```

**Review by bzkrouse — APPROVED** [review]
Thank you!!

**Pull request merged**